src/VerovioConverter.ts

```diff
@@ -18,14 +19,17 @@ import type { PlayerOptions } from './Player';
  */
 export class VerovioConverter
   extends VerovioConverterBase
-  implements IMIDIConverter
-{
+  implements IMIDIConverter {
   protected _vrv?: VerovioToolkitFixed;
   protected _timemap: MeasureTimemap = [];
   protected _midi?: ArrayBuffer;
   protected _options: VerovioOptionsFixed;
+  protected _xsltProcessor: IXSLTProcessor;
 
-  constructor(options?: VerovioOptionsFixed) {
+  constructor(
+    xsltProcessor: IXSLTProcessor,
```

Comment:
As per my comment in `Player.ts`, we don't need the client to instantiate the converter/renderer with the instance of `IXSLTProcessor`. It will be passed during `initialize` in the `options`.